core/password.go

```diff
@@ -0,0 +1,30 @@
+/*******************************************************************************
+ * Copyright 2018 Dell Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ * @author: Tingyu Zeng, Dell
+ * @version: 1.0.0
+ *******************************************************************************/
+
+package main
+
+import (
+	"github.com/sethvargo/go-password/password"
```

Comment:
Has anyone performed a review of this code? We're relying on this library to produce cryptographically secure passwords so it should definitely be vetted. If no one has vetted this library or plans to vet it, then we should use the go std library's random crypto functionality directly to generate such passwords.

Reply:
Looking at go-password it seems to implement features that we don't really need, like being able satisfy password strength rules (# of lowercase, # of digits, # of specials, etc).  Seems that generating a cryptorandom 256 bits and hex or base-64 encoding it would be sufficient and could be done inline without requiring a library and risking a supply chain attack on the password generator.

Reply:
we already removed the implementation of the password generation but leave this as an issue to be fixed in the future due to lack of resource.